spring-cloud-gcp-config/README.md

```diff
@@ -0,0 +1,13 @@
+# Spring Cloud Config Client integration for Google Cloud Runtime Configurator API
```

Comment:
Sorry for the annoyance but we're going with asciidoc for READMEs. I think that's a Spring standard. Can you convert this?